chat-components/src/components/footer/subcomponents/AudioNotificationButton.tsx

```diff
@@ -72,7 +72,9 @@ function AudioNotificationButton(props: ICommandButtonProps) {
             disabled={disabled}
             styles={iconButtonStyles}
             className={props.className}
-            title= {undefined}
+            title={muted ?
+                props.toggleAriaLabel ?? AriaLabels.TurnSoundOn :
```

Comment:
conceptually, this is a problem we have in OC-LCW, by using hardcoded texts, we are no supporting translation for screen readers, I have created this bug to discuss the problem -> https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3663322